model/schema/sssom.yaml

```diff
@@ -93,6 +85,11 @@ slots:
         description: (The CURIE of the Uberon term for "anatomical entity".)
       - value: biolink:Gene
         description: (The CURIE of the biolink class for genes.)
+  subject_type:
```

Comment:
name could be confused with the entity type (like disease or protein)

Reply:
I thought of that, but I guess this would be the case with these kinds of words no matter what. It was previously "type" (the state before this change). Any idea for an alternative? I kinda thought: rdf:type, so... `type`. At least some people wont be confused :)

Reply:
Can you please provide an example row in an SSSOM TSV that would use these entries in the PR description?

Reply:
Even in rdf terms it could be confusing. I am both rdf type namedindividual and Homo sapiens. I think the name is ok if sufficiently documented though, and besides. An alternative is owl_type

Reply:
so the goal here is to put verrry high level entity types? If that's the case, can we have an enumeration of a very small set of valid ones? or is this just a place where we expect people to put a slurry of things and then later we have to write tools to deconvolute it?

Reply:
@cmungall according to @cthoyt I restricted the set of legal `types`. What is your opinion on this?

Reply:
(also on _how_ I did it)

Reply:
Thanks Nico, this is now much easier to understand. Entity type could be inferred from external resources that aren't SSSOM and this could be considered scope creep, but also that's not always so easy to do, so having it explicit could be nice. At least I think the proposed update is better than what was there before.

Reply:
Exactly. This is purely optional metadata, which can be used by sssom-py for certain assumptions, like how to render an "OWL" or a "SKOS" version of the mapping. We do not always know the source, or have a standard mechanism to determine just based on the subject ID.

Reply:
I think this is a good and practical set of enums

If we are doing this I think we may need to do some work on specifying the owl mapping

E.g. how do I interpret A subClassOf B in the following scenarios:

- A is punned (subject_type: [Class, Individual]
- A is an individual only
- A is an individual B is a class
- A and B are a mix of OWL and SKOS types

These are pretty obscure edge cases but nevertheless we should document